src/components/Chat/InputMessageV2.tsx

```diff
@@ -369,6 +365,7 @@ const InputMessageV2 = ({
 
   const toggleChange = () => {
     if (messageDisable) {
+      // TODO: Garry, need to change this color
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

This TODO comment indicates a pending task. It's important to track these tasks and ensure they're completed before merging the PR. If the color change is significant, consider creating a separate task or issue to track it.

```diff
+ // TODO: Garry, need to change this color
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->